apps/server/src/infra/tsp-client/tsp-client.module.ts

```diff
@@ -1,7 +1,9 @@
+import { OauthModule } from '@modules/oauth';
```

Comment:
```suggestion
import { OauthModule } from '@modules/oauth/oauth.module';
```

Reply:
I added the OauthModule to the `index.ts`. As it was previously.